html/laravel/app/Http/Controllers/TaskController.php

```diff
@@ -147,19 +148,28 @@ public function show(Project $project, Task $task)
      * @param  \App\Models\Task  $task
      * @return \Illuminate\Http\Response
      */
-    public function edit(Project $project, Task $task)
+    public function edit(Project $project, Task $task, Comment $comment)
     {
         $task_kinds = TaskKind::all();
         $task_statuses = TaskStatus::all();
         $task_categories = TaskCategory::all();
         $assigners = User::all();
+        $comments = Comment::select([
+            'comments.*',
+            'users.name'
+            ])
+            ->from('comments')
+            ->join('users', 'comments.comment_user_id', '=', 'users.id')
+            ->where('comments.task_id', '=', $task->id)
+            ->get();
```

Comment:
リレーションメソッドのクエリビルダで書きなおしましょう。
```suggestion
        $comments = $task->task_comments()->with('user')->get();
```